docs/en/create-command.md

```diff
@@ -0,0 +1,427 @@
+```
+
+The result of the above example is:
+
+```php
+array(3) {
+  [0]=>
+  array(6) {
+    ["id"]=>
+    string(1) "1"
+    ["email"]=>
+    string(17) "user1@example.com"
+    ["name"]=>
+    string(5) "user1"
+    ["address"]=>
+    string(8) "address1"
+    ["status"]=>
+    string(1) "1"
+    ["profile_id"]=>
+    string(1) "1"
+  }
+  [1]=>
+  array(6) {
+    ["id"]=>
+    string(1) "2"
+    ["email"]=>
+    string(17) "user2@example.com"
+    ["name"]=>
+    string(5) "user2"
+    ["address"]=>
+    string(8) "address2"
+    ["status"]=>
+    string(1) "1"
+    ["profile_id"]=>
+    NULL
+  }
+  [2]=>
+  array(6) {
+    ["id"]=>
+    string(1) "3"
+    ["email"]=>
+    string(17) "user3@example.com"
+    ["name"]=>
+    string(5) "user3"
+    ["address"]=>
+    string(8) "address3"
+    ["status"]=>
+    string(1) "2"
+    ["profile_id"]=>
+    string(1) "2"
+  }
+}
+```
+
+Example 2: Fetching a single row from a table using `Yiisoft\Db\Command\CommandInterface::queryOne()`, return array the first row (in terms of an array) of the query result. Null is returned if the query results in nothing.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+$command = $db->createCommand('SELECT * FROM customer')->queryOne();
+```
+
+The result of the above example is:
+
+```php
+array(6) {
+  ["id"]=>
+  string(1) "1"
+  ["email"]=>
+  string(17) "user1@example.com"
+  ["name"]=>
+  string(5) "user1"
+  ["address"]=>
+  string(8) "address1"
+  ["status"]=>
+  string(1) "1"
+  ["profile_id"]=>
+  string(1) "1"
+}
+```
+
+Example 3: Fetching a single column from a table using `Yiisoft\Db\Command\CommandInterface::queryColumn()`, return array the values of the first column in the query result. An empty array is returned if the query results in nothing.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+
+$command = $db->createCommand('SELECT * FROM customer')->queryColumn();
+```
+
+The result of the above example is:
+
+```php
+array(3) {
+  [0]=>
+  string(1) "1"
+  [1]=>
+  string(1) "2"
+  [2]=>
+  string(1) "3"
+}
+```
+
+Example 4: Fetching a single value from a table using `Yiisoft\Db\Command\CommandInterface::queryScalar()`, return the value of the first column in the first row of the query result. False is returned if there is no value.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+
+$command = $db->createCommand('SELECT * FROM customer')->queryScalar();
+```
+
+The result of the above example is:
+
+```php
+string(1) "1"
+```
+
+Example 5: Fetching a single row using `Yiisoft\Db\Command\CommandInterface::query()`, return `Yiisoft\Db\DataReader\DataReaderInterface` object. You can iterate over the returned object to get the data.
```

Comment:
How about checking Yii2 guide about queries? It had a quite good structure and a very short but more than enough description: https://www.yiiframework.com/doc/guide/2.0/en/db-dao#executing-sql-queries